### ISSUE TITLE
Have you tried CRF boundary loss when training segmentation model

### ISSUE THREAD

**Issue by yaoqi-zd** [PengtaoJiang/OAA#3]
Hi, I find that some of the other WSSS paper used CRF boundary loss when training the semantic segmentation model with pseudo labels. Have you tried this loss function, or you tried but without performance benifits ?

**Comment by PengtaoJiang** [comment]
@yaoqi-zd   I have not tried the CRF boundary loss. In my opinion, if you use the crf boundary loss during the training step, the crf in the post-processing process may not be useful. You can try this loss function. If you have any results or questions, feel free to contact me.
